ui/src/app/components/webcomponents/action-step-form.component.ts

```diff
@@ -1356,8 +1347,7 @@ export class ActionStepFormComponent extends BaseComponent implements OnInit {
     for (let i = 0; i < testDataPlaceHolders.length; i++) {
       const content = testDataPlaceHolders[i].innerHTML;
       if ((testDataPlaceHolders[i]?.contentEditable||Boolean(isReplacer)) && ((testDataPlaceHolders[i].getAttribute("data-test-data-type") == null) ||
-        ((testDataPlaceHolders[i].getAttribute("data-test-data-type") != null && (Boolean(isReplacer) ||content == '@| |' || content == '!| |' || content == '*| |'
-          || content == "$| <span class='test_data_place'></span> |" || content == "~| <span class='test_data_place'></span> |" || content == "" || !this.removeHtmlTags(testDataPlaceHolders[i].textContent).trim().length))))) {
+        ((testDataPlaceHolders[i].getAttribute("data-test-data-type") != null)))) {
         testDataPlaceHolders[i].innerHTML = dataName;
```

Comment:
i removed this part because, checking content with above specified conditions and making an update to the testdata is not possible if it is already contains some data.

Reply:
tested without theese conditions they are working fine as expected